kotlinx-coroutines-core/build.gradle.kts

```diff
@@ -187,9 +223,11 @@ fun Test.configureJvmForLincheck(segmentSize: Int = 1) {
     minHeapSize = "1g"
     maxHeapSize = "4g" // we may need more space for building an interleaving tree in the model checking mode
     // https://github.com/JetBrains/lincheck#java-9
-    jvmArgs = listOf("--add-opens", "java.base/jdk.internal.misc=ALL-UNNAMED",   // required for transformation
+    jvmArgs = listOf(
+        "--add-opens", "java.base/jdk.internal.misc=ALL-UNNAMED",   // required for transformation
         "--add-exports", "java.base/sun.security.action=ALL-UNNAMED",
-        "--add-exports", "java.base/jdk.internal.util=ALL-UNNAMED") // in the model checking mode
+        "--add-exports", "java.base/jdk.internal.util=ALL-UNNAMED"
+    ) // in the model checking mode
```

Comment:
Pro tip: use Difftastic when reviewing such changes! In a structural diff, the trivial formatting edits don't show up at all.